src/harness/reference_models/interference/test_data/gwpz_1.json

```diff
@@ -1,13 +1,16 @@
-{  
-   "id":"incumbent/uls/1",
-   "type":"PART_90",
-   "deploymentParam":{  
-      "operationParam":{  
-         "operationFrequencyRange":{  
-            "lowFrequency":3650000000.0,
-            "highFrequency":3700000000.0
+{
+   "record": {  
+      "id":"incumbent/uls/1",
+      "type":"PART_90",
+      "deploymentParam": [
+      {  
+            "operationParam":{  
+               "operationFrequencyRange":{  
+                  "lowFrequency":3650000000.0,
+                  "highFrequency":3670000000.0
```

Comment:
The original "highFrequency" was 3700000000.0. Will this value change affect any test cases ? better keep it same as before.